.mergify.yml

```diff
@@ -51,10 +62,3 @@ pull_request_rules:
     actions:
       delete_head_branch: {}
 
-  - name: auto add wip
-    conditions:
-      # match a few flavours of wip
-      - title~=^(\[wip\]( |:) |\[WIP\]( |:) |wip( |:) |WIP( |:)).*
-    actions:
-      label:
-        add: ["status:block-merge"]
```

Comment:
This rules is not working and we also don't use WIPs PRs that much. 

We can remove start removing it.